### DISCUSSION TITLE
Minimize to Tray

### DISCUSSION THREAD

**Discussion by cariyawa** [TriliumNext/Trilium#3359] [General]
I am on Debian 11/ KDE. Is there are a way/ setting to minimize Trilium to system-tray when minimizing the app? I can do that by right clicking on the tray icon and choose hide, but it would be great it can be directly minimized to tray.

**Comment by zadam** [comment]
Hi, there isn't an option to do that automatically.

**Reply by cariyawa** [reply]
Thank you for the quick reply. It could be a great feature to add in tough :-)

**Reply by ghost** [reply]
I second that feature request.  I'm on Windows.
Some tools (Joplin, for instance) allow to : 
- start minimized in the system tray
- display icon in the system tray
Also an option to "minimize to the system tray" when clicking on the "close" button (instead of killing the app) would be great too.

**Comment by ghost** [comment]
The feature request is actually already open : #3743
